tests/Feature/Router/RouterMatchTest.php

```diff
@@ -26,18 +26,6 @@ public function test_respond_when_everything_matches(): void
         });
     }
 
-    public function test_not_respond_when_the_uri_does_not_matches(): void
-    {
-        $_SERVER['REQUEST_URI'] = 'https://example.org/unexpected/uri';
-        $_SERVER['REQUEST_METHOD'] = 'GET';
-
-        $this->expectOutputString('');
-
-        Router::configure(static function (Routes $routes): void {
-            $routes->get('other/uri', FakeController::class, 'basicAction');
-        });
-    }
-
     public function test_not_respond_when_the_method_does_not_matches(): void
```

Comment:
You also forgot this test, I think you should move it to ErrorHandlingTest as well.

Reply:
And also the test called test_match_does_not_matches_other_methods